odl/trafos/wavelet.py

```diff
@@ -525,71 +585,82 @@ def __init__(self, domain, nscales, wbasis, pad_mode):
         nscales : int
             Number of scales in the coefficient list.
             The maximum number of usable scales can be determined
-            by ``pywt.dwt_max_level``. For more information see
+            by `pywt.dwt_max_level`. For more information see
             the corresponding `documentation of PyWavelets
             <http://www.pybytes.com/pywavelets/ref/\
 dwt-discrete-wavelet-transform.html#maximum-decomposition-level\
 -dwt-max-level>`_ .
 
-        wbasis :  {string, ``pywt.Wavelet``}
-            If a string is given, converts to a ``pywt.Wavelet``.
+        wbasis :  {string, `pywt.Wavelet`}, optional
+            If a string is given, converts to a `pywt.Wavelet`.
             Describes properties of a selected wavelet basis.
             See PyWavelet `documentation
             <http://www.pybytes.com/pywavelets/ref/wavelets.html>`_
 
             Possible wavelet families are:
 
-            Haar (``haar``)
+            'Haar': Haar
+
+            'db'  : Daubechies
```

Comment:
Help pages have a non-proportional font typically, so the extra spaces look wrong there.